docs/adr/0038-logic-for-displaying-attachment-types-in-emails.md

```diff
@@ -24,8 +24,11 @@ Brief the logic flows to make it easier to track changes during `attachment` dis
 2. Inline attachments: Displayed within the email body
 
 - Display is only allowed when the following conditions are met:
-  - `cid is not NULL` AND `disposition = 'inline'`
+  - `cid is not NULL` AND `disposition = 'inline' || disposition = NULL`
```

Comment:
with `disposition is empty`?

Reply:
`ContentDisposition` is `enum`, there's no `empty` value.